crates/forge-runner/src/printing.rs

```diff
@@ -49,5 +49,12 @@ pub(crate) fn print_test_result(test_result: &TestCaseSummary) {
         }
     };
 
-    println!("{result_header} {result_name}{fuzzer_report}{block_number_message}{result_message}");
+    let gas_report = match test_result {
+        TestCaseSummary::Passed { gas, .. } => {
+            format!("\nGas used: {gas}\n")
+        }
+        _ => String::new(),
+    };
+
+    println!("{result_header} {result_name}{fuzzer_report}{block_number_message}{result_message}{gas_report}");
```

Comment:
This is fine as a temporary thing, but eventually forge cli should probably handle the UI side of gas calculation (introduce some configuration for showing it or not etc.).

Reply:
So I think I will remove it for now and create an issue in forge-cli. Are you fine with it?

Reply:
Seems good to me

Reply:
Actually we have an issue for that
https://github.com/orgs/foundry-rs/projects/3/views/4?pane=issue&itemId=37728246